src/components/MDX/Card.styled.ts

```diff
@@ -0,0 +1,24 @@
+import {styled} from "@styles/stitches.ts";
+
+const CardStyled = styled("div", {
```

Comment:
Prefix this so that it's not confused with the Card component used elsewhere?

```suggestion
const MDXCardStyled = styled("div", {
```